bindata/scripts/enable-rdma.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+REDHAT_RELEASE_FILE="/etc/redhat-release"
+
+function kmod_isloaded {
+  if grep --quiet '\(^ib\|^rdma\)' <(chroot /host/ lsmod); then
+    echo "RDMA kernel modules loaded"
+    true
+  else
+    echo "RDMA kernel modules not loaded"
+    false
+  fi
+}
+
+function trigger_udev_event {
+  echo "Trigger udev event"
+  chroot /host/ modprobe -r mlx4_en && chroot /host/ modprobe mlx4_en
```

Comment:
This is dangerous, if the default interface on the host is using mlx4_en or mlx5_core driver, then the worker node will lose connection with openshift masters.

Reply:
hmm, that is a problem. However, since connect will go back immediately and this activity will only be triggered one time, I think that is acceptable.

Reply:
The link state can go back soon, not sure if all other network configuration on that interface can be properly restored.
if yes, we can fix it later.